contracts/LifToken.sol

```diff
@@ -267,8 +274,23 @@ contract LifToken is Ownable, ERC20, SafeMath, PullPayment {
 
     }
 
+    // Function that allows an address to claim a futurePayment on tokens
+    function claimTokensPayment(uint pos) external onStatus(4,0) {
+
+      if ((futurePayments[pos].tokens == 0) || (futurePayments[pos].owner != msg.sender) ||
+        ((futurePayments[pos].afterBlock > 0) && (futurePayments[pos].afterBlock > block.number)))
+        throw;
+
+      uint formatedBalance = safeMul(futurePayments[pos].tokens, LONG_DECIMALS);
+
+      totalSupply = safeAdd(totalSupply, futurePayments[pos].tokens);
+      balances[msg.sender] = safeAdd(balances[msg.sender], formatedBalance);
+      futurePayments[pos].tokens = 0;
+
+    }
+
     // Function that allows the owner to distribute the tokens after a crowdsale
-    function distributeTokens(uint stage, address buyer, bool discount) external onStatus(4,0) onlyOwner() {
+    function distributeTokens(uint stage, address buyer, bool discount) external onStatus(4,0) {
```

Comment:
so now this function can be called by anyone, not only by the owner? So each buyer could claim his tokens by calling this function?